components/tally/tally.app.mjs

```diff
@@ -1,11 +1,74 @@
+import { axios } from "@pipedream/platform";
+
 export default {
   type: "app",
   app: "tally",
-  propDefinitions: {},
+  propDefinitions: {
+    forms: {
+      label: "Form",
+      description: "Select a form",
+      type: "string",
+      async options() {
+        const forms = await this.getForms();
+
+        return forms.map((form) => ({
+          label: form.name,
+          value: form.id,
+        }));
+      },
+    },
+  },
   methods: {
-    // this.$auth contains connected account data
-    authKeys() {
-      console.log(Object.keys(this.$auth));
+    _accessToken() {
+      return this.$auth.oauth_access_token;
+    },
+    _apiUrl() {
+      return "https://api.tally.so";
+    },
+    async _makeRequest(path, options = {}, $ = undefined) {
+      return await axios($ ?? this, {
+        url: `${this._apiUrl()}/${path}`,
+        headers: {
+          Authorization: `Bearer ${this._accessToken()}`,
+        },
+        ...options,
+      });
+    },
+    async _createWebhook(data) {
+      return await this._makeRequest("webhooks", {
+        method: "post",
+        data: {
+          ...data,
+          externalSubscriber: "PIPEDREAM",
+        },
+      });
+    },
+    async _removeWebhook(webhookId) {
+      return await this._makeRequest(`webhooks/${webhookId}`, {
```

Comment:
`return await` [is not necessary](https://eslint.org/docs/rules/no-return-await)

```suggestion
      return this._makeRequest(`webhooks/${webhookId}`, {
```